src/main/java/org/opensearch/sdk/sample/helloworld/HelloWorldExtension.java

```diff
@@ -34,6 +37,7 @@ public class HelloWorldExtension extends BaseExtension {
      * Optional classpath-relative path to a yml file containing extension settings.
      */
     private static final String EXTENSION_SETTINGS_PATH = "/sample/helloworld-settings.yml";
+    private static final String FORBIDDEN_REGEX = "forbidden";
```

Comment:
Please put all the new setting code in a new class, not the top level extension.   The `ExampleCustomSettingsConfig` in the plugin we're modeling is a good class name unless you have a better idea.

In this class, you will add the `getSettings()` extension point referencing this setting in the new class.

Reply:
ACK